extensions/adobe/experience/customerJourneyManagement/subscription.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/customerJourneyManagement/subscription",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe ExperienceEvent - Subscription details",
+  "description": "Details about subscription to subscription list",
+  "type": "object",
+  "meta:extensible": true,
+  "definitions": {
+    "subscription": {
+      "properties": {
+        "xdm:subscriptionListId ": {
```

Comment:
@prabhum2 / @mjasrotia : Can you guide when we should be using the xdm namespace or solution specific namespaces while defining the fields in mixins.

Reply:
You are trying to declare a new Datatype here. If this datatype is specifically for "adobe.com/experience/customerJourneyManagement" use cases (which the $id implies it is) then all these fields should be assigned to that namespace. 

If this was intended to be a generic standard datatype under the "adobe.com/xdm" namespace, then the fields would fall under the "xdm" namespace. 

We want to avoid creating datatypes if they are never intended to be re-used for multiple fields of that type. I see where we are using this datatype in your mixin above for one field, but if its that field only we should simply nest these fields under it.